api/wasm.go

```diff
@@ -112,6 +117,8 @@ func ValueTypeName(t ValueType) string {
 		return "f32"
 	case ValueTypeF64:
 		return "f64"
+	case ValueTypeVector:
```

Comment:
ValueTypeV128 would be the most common naming convention though internally some use i128